lib/dal/tests/integration_test/module.rs

```diff
@@ -169,93 +167,6 @@ async fn module_export_simple(ctx: &mut DalContext) {
     assert_eq!(exported_pkg_func_names, expected_func_names);
 }
 
-#[test]
-async fn dummy_sync(ctx: &DalContext) {
```

Comment:
This test was already a bit iffy in what it was testing, but now it especially makes less sense without creating a bytestream of "pkg data" and inserting a cached module row. We'll delete it for now since its coverage would be questionable without more test helper investment.